### PR TITLE
[CloudPC]Add gallery image entity and related APIs

### PR DESCRIPTION
**Pull request by AshleyYangSZ**

### CODE REVIEW

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+doc_type: resourcePageType
+---
+
+# cloudPcGalleryImage resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the gallery image resource on Cloud PC.
```
[the remark this answers is not in the document]
Comment (AshleyYangSZ) [reply api-reference/beta/resources/cloudpcgalleryimage.md:16]: Updated the description. Please help review again.

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|The official display name of the gallery image. Read-only.|
+|endDate|Date|The date in which this image is no longer within long-term support. The Cloud PC will continue to provide short-term support. Read-only.|
+|expirationDate|Date|The date when the image is no longer available. Read-only.|
+|id|String|Unique identifier for the gallery image resource on the Cloud PC. Read-only.|
+|offer|String|The offer name of the gallery image. This value will be passed to Azure to get image resource. Read-only.|
+|offerDisplayName|String|The official display offer name of the gallery image. For example, Windows 10 Enterprise + OS Optimizations. Read-only.|
+|publisher|String|The publisher name of the gallery image. This value will be passed to Azure to get image resource. Read-only.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcgalleryimage.md:35]: Please, add "the" before image. Suggested revision:

- ...to Azure to get the image resource...

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|The official display name of the gallery image. Read-only.|
+|endDate|Date|The date in which this image is no longer within long-term support. The Cloud PC will continue to provide short-term support. Read-only.|
+|expirationDate|Date|The date when the image is no longer available. Read-only.|
+|id|String|Unique identifier for the gallery image resource on the Cloud PC. Read-only.|
+|offer|String|The offer name of the gallery image. This value will be passed to Azure to get image resource. Read-only.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcgalleryimage.md:33]: Please, add "the" before image. Suggested revision:

- ...to Azure to get the image resource...

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+|offerDisplayName|String|The official display offer name of the gallery image. For example, Windows 10 Enterprise + OS Optimizations. Read-only.|
+|publisher|String|The publisher name of the gallery image. This value will be passed to Azure to get image resource. Read-only.|
+|recommendedSku|String|Recommended Cloud PC SKU for this gallery image. Read-only.|
+|sizeInGB|Int32|The size of this image in gigabytes. Read-only.|
+|sku|String|The SKU name of the gallery image. This value will be passed to Azure to get image resource. Read-only.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcgalleryimage.md:38]: Please, add "the" before image. Suggested revision:

- ...to Azure to get the image resource...

File: api-reference/beta/api/virtualendpoint-list-galleryimages.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
+---
+title: "List cloudPcGalleryImages"
```
Comment (JarbasHorst) [inline api-reference/beta/api/virtualendpoint-list-galleryimages.md:2]: Please, update the title field to be "List galleryImages". Please, check [this ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/14396#discussion_r733099140)comment for more details. Suggested revision:

- title: "List galleryImages"

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
+---
+title: "cloudPcGalleryImage resource type"
+description: "Represents the gallery image resource of current organization."
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcgalleryimage.md:3]: Please, add "the" before current organization.

File: api-reference/beta/resources/cloudpcgalleryimage.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+doc_type: resourcePageType
+---
+
+# cloudPcGalleryImage resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the gallery image resource of current organization which can be used to provision a Cloud PC.
```
Comment (JarbasHorst) [inline api-reference/beta/resources/cloudpcgalleryimage.md:16]: Please, add "the" before current organization.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**